src/app/datagrid/expandable-rows/expandable-rows.html

```diff
@@ -0,0 +1,87 @@
+            <label for="replace">Replace row</label>
+        </div>
+        <div class="checkbox-inline">
+            <input type="checkbox" name="fixed-height" id="fixed-height" [(ngModel)]="fixedHeight">
+            <label for="fixed-height">Fixed height</label>
+        </div>
+        <div class="checkbox-inline">
+            <input type="checkbox" name="selectable" id="selectable" [(ngModel)]="selectable">
+            <label for="selectable">Selectable</label>
+        </div>
+        <div class="checkbox-inline">
+            <input type="checkbox" name="slow-load" id="slow-load" [(ngModel)]="slowLoad">
+            <label for="slow-load">Slow load</label>
+        </div>
+    </div>
+</form>
+
+<clr-datagrid [style.height.px]="fixedHeight ? 458 : null" [(clrDgSelected)]="selected">
+    <clr-dg-column>User ID</clr-dg-column>
+    <clr-dg-column [clrDgField]="'name'">Name</clr-dg-column>
+    <clr-dg-column [clrDgField]="'creation'">Creation date</clr-dg-column>
+    <clr-dg-column [clrDgField]="'pokemon.name'">Pokemon</clr-dg-column>
+    <clr-dg-column [clrDgField]="'color'">Favorite color</clr-dg-column>
+
+    <clr-dg-row *clrDgItems="let user of users" [clrDgItem]="user">
+        <clr-dg-cell>{{user.id}}</clr-dg-cell>
+        <clr-dg-cell>{{user.name}}</clr-dg-cell>
+        <clr-dg-cell>{{user.creation | date}}</clr-dg-cell>
+        <clr-dg-cell>{{user.pokemon.name}}</clr-dg-cell>
+        <clr-dg-cell>
+            <span class="color-square" [style.backgroundColor]="user.color"></span>
+        </clr-dg-cell>
+
+        <!-- Example using a wrapper component -->
+        <!--<detail-wrapper *clrIfExpanded ngProjectAs="clr-dg-row-detail" class="datagrid-row-flex"></detail-wrapper>-->
+
+        <clr-dg-row-detail *clrIfExpanded [clrDgReplace]="replace">
+            <template [clrFakeLoader]="slowLoad" clrLoading>
+                <template [ngIf]="detail === 'default'">
+                    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin in neque in ante placerat mattis id sed
+                    quam. Proin rhoncus lacus et tempor dignissim. Vivamus sem quam, pellentesque aliquet suscipit eget,
+                    pellentesque sed arcu. Vivamus in dui lectus. Suspendisse cursus est ac nisl imperdiet viverra. Aenean
+                    sagittis nibh lacus, in eleifend urna ultrices et. Mauris porttitor nisi nec velit pharetra porttitor.
+                    Vestibulum vulputate sollicitudin dolor ut tincidunt. Phasellus vitae blandit felis. Nullam posuere
+                    ipsum tincidunt velit pellentesque rhoncus. Morbi faucibus ut ipsum at malesuada. Nam vestibulum felis
+                    sit amet metus finibus hendrerit. Fusce faucibus odio eget ex vulputate rhoncus. Fusce nec aliquam leo,
+                    at suscipit diam.
+                </template>
+
+                <template [ngIf]="detail === 'columns'">
```

Comment:
If you are merging to master, you're going to want to change these to `ng-template`